### DISCUSSION TITLE
Possbility to use RS485 to WiFi gateway (elfin ew11)

### DISCUSSION THREAD

**Discussion by shavermcspud** [flip555/seplos_bms_ha#54] [Seplos BMS Development Talk & Ideas]
is there a way or plan to be able to use an rs485 gateway to access the BMS rather than a com port as that seems the only option.

**Comment by flip555** [comment]
Yes it should be possible just trying to iron out some of the seplos stuff then it should be pretty straight forward to integrate.

I'd be looking at integrating a few different connection methods as they'd all be useful in my larger project which will handle other similar things

**Reply by shavermcspud** [reply]
Many thanks for the reply. Its been working really well for me so many thanks for your hard work.

**Comment by biggio85** [comment]
taking up the topic again. So is it possible or not? Is it enough to enter the IP address of the Elfin once configured?

**Comment by lukaweb** [comment]
I think this is an important feature, since many users work with wi-fi elfin on lan
